scripts/proposals/AIP3/verifyDeployment.ts

```diff
@@ -0,0 +1,37 @@
+import fs from "fs";
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { AIP3Action__factory } from "../../../typechain-types";
+import { assertEquals } from "../../testUtils";
+import dotenv from "dotenv";
+dotenv.config();
+
+const { actionAddress } = JSON.parse(
+  fs.readFileSync("./scripts/proposals/AIP3/data/42161-AIP3-data.json").toString()
+);
+
+const ARB_URL = process.env.ARB_URL;
+if (!ARB_URL) throw new Error("ARB_URL required");
+const expectedAddressRegistry = "0x56C4E9Eb6c63aCDD19AeC2b1a00e4f0d7aBda9d3";
+const expectedConstitutionHash =
```

Comment:
I get a different constitution hash in https://github.com/ArbitrumFoundation/docs/pull/597

Reply:
nvm got the correct hash now